src/docs/sdk/features.mdx

```diff
@@ -396,6 +396,59 @@ The `fingerprints` field should be set to `["$operationName", "$operationType",
 }
 ```
 
+## GraphQL Performance
+
+The GraphQL Performance integration should match the guidelines for [GraphQL Client Integrations](/sdk/features/#graphql-client-integrations) with a few differences:
+
+The transaction's name should be set with the GraphQL operation name, if possible, otherwise fallback to something unique that makes sense, e.g. the canonical name of the actual/generated class.
+
+The transaction's description should be set with the GraphQL operation name, operation type and status code, if possible.
+
+The `request.api_target` should be set with `graphql`.
+
+The `request.data` should be set with the raw GraphQL request payload, if possible.
+
+The `contexts.response.data` should be set with the raw GraphQL response payload, only if there were `errors`.
+
+Spans should be created for [resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/), if possible.
+
+Spans should be created for [data loaders](https://graphql.org/learn/best-practices/#server-side-batching-caching/), if possible.
```

Comment:
Assuming we add keys used as input for the data loader. How do we serialize non primitive keys that could bloat events?

Reply:
@adinauer I'd fall back to `.toString` as best effort.

Reply:
Sounds OK but could still bloat if the language simply puts everything into `toString()` - so something to be aware of.

For Java it's a compromise between risking bloat (just put the objects in the breadcrumb and let serialization handle it) and having useless data like `MyClass@12345` (by using `toString`). Let's go with `toString` for now. We can always change if we come up with something better.

Reply:
@adinauer maybe the best decision is to not add such values then, risking the event to be dropped or having useless data does not feel good.

Reply:
I think there are cases where this data is very valuable. The useless `toString` output only applies to complex keys. For simple keys it should be fine. So I vote for adding keys.